practice/src/index.html

```diff
@@ -0,0 +1,164 @@
+          <div class="monthly-payment__min-amount">
+            <h2 class="result__title">From</h2>
+            <span id="monthly-payment-min-amount" class="result">0 VND</span>
+          </div>
+          <div class="monthly-payment__max-amount">
+            <h2 class="result__title">To</h2>
+            <span id="monthly-payment-max-amount" class="result">0 VND</span>
+          </div>
+        </div>
+        <div class="monthly-payment__total-iterest-payable">
+          <h2 class="result__title">Total interest payable</h2>
+          <span class="result" id="total-interest-payable">0 VND</span>
+          <button class="btn-open-modal" id="button-export">See more</button>
+        </div>
+
+      </div>
+    </div>
+  </div>
+
+
+  <!-- modal -->
+  <div class="modal-container">
+    <div class="table-result">
+      <div class="table-result__header">
+        <button class="header__cancel-button"  id="button-export">
+          <img src="./assets/icons/x_icon.svg" alt="cancel-icon"></button>
+        <h1 class="table-result__title">Loan payment schedule sheet with decreasing balance</h1>
+      </div>
+      <div class="table-result__body">
+        <table class="table-result__table" id="modal-table-result">
+          <thead class="table__header-footer">
+            <th class="table__header-footer-content">#</th>
+            <th class="table__header-footer-content">REPAYMENT PERIOT</th>
+            <th class="table__header-footer-content">REMAINING ORIGINAL AMOUNT</th>
+            <th class="table__header-footer-content">ORIGIN</th>
+            <th class="table__header-footer-content">INTEREST</th>
+            <th class="table__header-footer-content">TOTAL PRINCIPAL + INTEREST</th>
+          </thead>
+          <tbody>
+          </tbody>
+          <tfoot>
+            <tr class="table__header-footer">
+              <th class="table__header-footer-content">TOTAL</th>
+              <th class="table__header-footer-content"></th>
+              <th class="table__header-footer-content"></th>
+              <th class="table__header-footer-content" id="total-origin">0</th>
+              <th class="table__header-footer-content" id="total-interest">0</th>
+              <th class="table__header-footer-content" id="total-payment">0</th>
+            </tr>
+          </tfoot>
+        </table>
+      </div>
+
+      <button class="button-export" id="export">Export to excel</button>
+
+    </div>
+  </div>
+  <script type="module" src="./js/views/page_load_view.js"></script>
+  <script type="module" src="./js/controllers/page_load_controller.js"></script>
+  <script type="module" src="./js/models/loan_payment_model.js"></script>
+  <script type="module" src="./js/models/loan_model.js"></script>
+  <script type="module" src="./js/views/form_input_view.js"></script>
+  <script type="module" src="./js/controllers/form_input_controller.js"></script>
+  <script type="module" src="./js/main.js"></script>
```

Comment:
Can you make it simple, just import ./js/main.js like css
Too many scripts here